Formula/lavinmq.rb

```diff
@@ -13,7 +13,9 @@ class Lavinmq < Formula
   depends_on "pcre2"
 
   def install
-    system "make", "all", "DOCS="
+    system "make", "bin/lavinmq", "bin/lavinmqperf", "DOCS="
+    system "crystal", "build", "src/lavinmqctl.cr", "-o", "bin/lavinmqctl", "--release", "--error-on-warnings",
+      "--link-flags=-pie"
```

Comment:
What is the difference when building it like this vs via `make`? I fail to spot a significant difference 😁

Reply:
Nevermind, omitting ` -Dgc_none` was key :)